prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -1384,6 +1534,10 @@ void rxr_pkt_handle_eager_rtw_recv(struct rxr_ep *ep,
 		return;
 	}
 
+	desc = rx_entry->desc[0];
+	ofi_copy_to_hmem_iov(desc ? desc->peer.iface : FI_HMEM_SYSTEM,
```

Comment:
What if the iov in the write request is to a CUDA buffer? Doesn't this also need to be copied via fi_read?